OrbitGl/TimeGraph.cpp

```diff
@@ -763,10 +763,12 @@ void TimeGraph::DrawTracks(GlCanvas* canvas, PickingMode picking_mode) {
         std::string process_name = GOrbitApp->GetCaptureData().process_name();
         thread_track->SetName(process_name);
         thread_track->SetLabel(process_name + " (all threads)");
+        thread_track->SetNumberOfPrioritizedTrailingCharacters(strlen("(all threads)"));
       } else {
         const std::string& thread_name = GOrbitApp->GetCaptureData().GetThreadName(tid);
         track->SetName(thread_name);
         std::string track_label = absl::StrFormat("%s [%u]", thread_name, tid);
+        thread_track->SetNumberOfPrioritizedTrailingCharacters((std::to_string(tid)).size() + 2);
```

Comment:
You are converting tid twice to string by different algorithms and expecting the result is the same. Probably it will, so not a big deal. But I think it would be nicer to create `auto tid_str = std::to_string(tid)` and use it in absl::StrFormat and in the .size-call. WDYT?

Reply:
Perfect, done!